EventOrganizer/src/EventOrganizer.Blazor/Pages/EventDetail.razor

```diff
@@ -1,45 +1,71 @@
 ﻿@page "/events/{id}"
 @if (Event != null)
 {
-    <Row>
-        <Column ColumnSize="ColumnSize.Is8">
-            <h1>@Event.Title</h1>
-            <p>Start time: @Event.StartTime.ToLongDateString()</p>
-            <p>@Event.Description</p>
+
+    <Row Class="mb-4">
+        <Column Class="text-left">
+            <h1>Event Detail</h1>
         </Column>
-        <Column ColumnSize="ColumnSize.Is4">
+        <Column Class="text-right">
+            <a href="/" Class="mt-2 btn btn-dark"><i class="fa fa-arrow-left"></i> Back</a>
```

Comment:
In nevest Blazorise rc1 we now have `<Button Type="ButtonType.Link" To="your url">`